### ISSUE TITLE
Handling of links in 5XX fields

### ISSUE THREAD

**Issue by nichtich** [scriptotek/mc2skos#31]
[5XX: See Also From Tracings](http://www.loc.gov/marc/authority/concise/ad550.html) is currently mapped to `skos:related` or `skos:broader` depending on `$w` but the value of `$w` can also indicate `skos:narrower` and other kinds of (non)relations. In particular subfield `$4` was added to directly specify relation with an URI if `$w` has value `r`. Possible values of `$w`:

~~~~
a - Earlier heading	h - Narrower term
b - Later heading	i - Reference instruction phrase in subfield $i
d - Acronym	        n - Not applicable
f - Musical composition	r - Relationship designation in $i or $4
g - Broader term	t - Immediate parent body
~~~~

**Comment by danmichaelo** [comment]
That I didn't add `skos:narrower` was just sloppiness.

For the case where `$w = "r"` and `$4` starts with `"http://"`, I guess we could add that URI as the triple predicate? Let me know if you have any example records using this.

(At least) the "a", "b" and "d" seems to be concept-term relations (or term-term) rather than concept-concept-relations, so perhaps it's better to use `skos:altLabel` for "a" and "d"? A problem being that there might be separate heading records that will be converted to their own skos concepts.. but hopefully the earlier headings will be marked as deprecated. To mark the relations between former headings and current ones, perhaps it would work to add `dcterms:isReplacedBy` and `dcterms:replaces`?

**Comment by nichtich** [comment]
I think we don't need to cover all these cases just "a", "b", "d", and "r" with URI in $4. I have not seen records with the latter yet, subfield `$4` was just added to the rules. I'd expect values such as "http://rdf-vocabulary.ddialliance.org/xkos#hasPart".

**Comment by danmichaelo** [comment]
Right, seems like URIs in `$4` was added in [MARC PROPOSAL NO. 2017-01](http://www.loc.gov/marc/mac/2017/2017-01.html) "Redefining Subfield $4 to Encompass URIs for Relationships in the MARC 21 Authority and Bibliographic Formats". I went ahead and added support for it while into it.

**Comment by nichtich** [comment]
When 5XX can also provide broader links we should disable use of `parent_notation` if the URI pattern is not based on `{object}` (see https://github.com/scriptotek/mc2skos/commit/cdf977fbe2088ff4ef64b780bf296ef17e4335c1).
